user-support/README.md

```diff
@@ -96,6 +96,10 @@ The exterminator is responsible for making sure a
 [release](https://github.com/windmilleng/tilt/blob/master/DEVELOPING.md#releasing)
 happens every non-holiday week.
 
+Each release should have a user-friendly [release notes section](https://github.com/windmilleng/tilt/releases)
+highlighting major changes, in addition to the automatically generated 
+changelog section.
```

Comment:
Yes! I think this section will be most useful to future Tilters with some specified intent, how about:
```suggestion
changelog section. The intent of these human-readable release notes is to communicate changes to our users, with special attention to anything that may break existing workflows, as well as exciting new features.
```

Reply:
@maiamcc : I've purposely avoided the term "human-readable", since I think the existing commit messages are meant for humans to read. But perhaps just for different humans.

I used "user-friendly" above. But maybe that's not an accurate term either. I like how you specified "breaking workflows" and "exciting new features", etc. And plan to incorporate those here.

Are you opposed if we avoid "human-readable" as a term?